.env.example

```
@@ -357,6 +357,14 @@ SUPABASE_WEBHOOK_SECRET=
 OWUI_BASE_URL=http://open-webui:8080
 OWUI_ADMIN_TOKEN=
 OWUI_E2E_MODE=false
+# Open WebUI native tool calling. "legacy" (the default in
+# deploy/docker/docker-compose.yml) stops Open WebUI attaching its builtin tool
+# schemas to every chat request: no demo route can serve them, OpenRouter
+# answers 404 "No endpoints found that support tool use" and the Groq free tier
+# answers 429 on its 6000 token per minute ceiling, since the schemas alone are
+# 3144 prompt tokens against 52 without them. Set it to native only when the
+# alias in use routes to a tool capable model.
+OWUI_DEFAULT_FUNCTION_CALLING=legacy
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OWUI_DEFAULT_FUNCTION_CALLING key should go before the OWUI_E2E_MODE key

(UnorderedKey)